src/main/java/uk/gov/hmcts/darts/transcriptions/exception/TranscriptionError.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.darts.transcriptions.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.darts.common.exception.DartsApiError;
+
+@Getter
+@RequiredArgsConstructor
+public enum TranscriptionError implements DartsApiError {
+    FAILED_TO_PROCESS_TRANSCRIPTION_REQUEST("100",
+                                    HttpStatus.INTERNAL_SERVER_ERROR,
+                                    "Failed to process transcription request");
```

Comment:
If you change this to be BAD_REQUEST - then please consider changing process to validate